### PR TITLE
fixing timestamp bug and removing user_info from track calls

### PR DESCRIPTION
**Pull request by usmanity**

### CODE REVIEW

File: lib/outbound.rb [hunk 1]
```diff
@@ -43,13 +43,13 @@ def Outbound.identify user_id, info={}
     return @ob.identify user_id, info
   end
 
-  def Outbound.track user_id, event, properties={}, user_info={}, timestamp
+  def Outbound.track user_id, event, properties={}, timestamp=nil
```
Comment (nburns) [inline lib/outbound.rb]: why no parens?

Comment (usmanity) [inline lib/outbound.rb]: following convention, but I'll add parens coz that's not a biggie

**Comment by usmanity** [comment]
updated

**Comment by nburns** [comment]
:+1:

**Pull request merged**